internal/pkg/plugin/manager.go

```diff
@@ -109,6 +120,15 @@ func (manager *Manager) _ReloadConfig(newConfig config.Config) error {
 }
 
 func (manager *Manager) _RegisterShutdownSignalHandlers() {
+
+	if (manager.ProfileFlag == "cpu") {
+		// PROFILER CPU profiling by default
+		ProfileCleaner = profile.Start(profile.NoShutdownHook)
+		// PROFILER MEMORY profiling by default
```

Comment:
wondering why we've got these comments here ?
`// PROFILER MEMORY profiling by default`